src/coprocessor/dag/storage/ranges_iter.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Note: this file is almost simply moved from the original `ranges_iter.rs`. However this new file no longer distinguish point range and interval range.